CHANGELOG.rst

```diff
@@ -1,10 +1,75 @@
 Changelog
 =========
 
-1.13.0 (2024-06-14)
+1.13.0 (2024-06-19)
+-------------------
 
 **New features**
-* [#1637] Added 2FA which can be disabled by the environment variable``DISABLE_2FA``.
+
+* API:
+  * [#1596] Added bulk import of documents (**NOTE** this change is not supported when CMIS is enabled)
+  * [#1587, #1630] Added new fields for ``Zaak` resource: ``created_on``, ``communicatiekanaal_naam``
```

Comment:
```suggestion
  * [#1587, #1630] Added new experimental fields for ``Zaak`` resource: ``created_on``, ``communicatiekanaal_naam``
```

If these fields are NOT part of the spec, I assume they are marked as experimental?

Reply:
`communicatiekanaal_naam` is exposed in API and marked as experimental in the OAS. `created_on` is not exposed in API, I'll adjust CHANGELOG

Reply:
Changed